Library/ViewModels/PaymentMethodsViewModelTests.swift

```diff
@@ -243,7 +255,8 @@ internal final class PaymentMethodsViewModelTests: TestCase {
     let apiService = MockService(deletePaymentMethodResult: .failure(.invalidInput))
     withEnvironment(apiService: apiService) {
 
-      self.vm.inputs.viewWillAppear()
+      self.vm.inputs.viewDidLoad()
```

Comment:
Can we keep the `viewWillAppear` calls in these tests? As we found out while pairing, it's often a good idea to ping the view model with the correct/expected lifecycle events so that we know they won't interfere with each other based on the outputs.